app/Repositories/Eloquents/BookEloquentRepository.php

```diff
@@ -28,8 +34,8 @@ public function find($id, $with = ['medias','categories'], $dataSelect = ['*'])
         $book = $this->model()->findOrFail($id);
 
         return $this->model()
-        ->with($with)
-        ->findOrFail($id);
+            ->with($with)
+            ->findOrFail($id);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>Expected 1 space between comma and "'categories'"; 0 found